internal/templates/internal/robots.txt

```diff
@@ -1,2 +1,2 @@
 User-agent: *
-Disallow: /*?
+Disallow: /
```

Comment:
What is the difference of letting the URLs being crawled here vs e.g. `web/`?

Reply:
Hm, do you mean if it said `Disallow: web/` instead of what it currently says..? 🤔

Reply:
Oh, I meant you added a disallowed: /*? at web/robots.txt and was curious by you added/removed the asterisk on certain pages. Hope it makes the question clearer :)

Reply:
Ah, I see 😅 So:
* in `internal/robots.txt`, I set `Disallow` to just `/`, as we should disallow crawling of *all* the pages of that subdomain (as it's not public), instead of only disallowing crawling of URLs with query parameters;
* in `admin/robots.txt`, the reason is the same as for `internal`;
* in `web/robots.txt`, I added `Disallow: /*?`, because we generally don't want crawling of query URLs, as they're most often created on a whim (as part of searching/filtering/sorting things on a page), and not something we want appearing among the search results of a search engine;
* in `docs/robots.txt`, I left the `Disallow` rule as it is, for the same reason as for `web`.

Did this clear things up? 😊